frontend/.env.local

```
@@ -115,3 +115,14 @@ NEXT_PUBLIC_OTEL_COLLECTOR_ENDPOINT=http://localhost:4318
 
 # I18N_LNG: Interface language ('en' or 'zh-CN')
 I18N_LNG=en
+
+# ============================================================================
+# Device Configuration (Local Device Feature)
+# ============================================================================
+
+# NEXT_PUBLIC_DEVICE_INSTALL_COMMAND: One-click install command for local device
+
+NEXT_PUBLIC_DEVICE_INSTALL_COMMAND=curl -fsSL https://raw.githubusercontent.com/wecode-ai/Wegent/main/install-wework.sh | bash
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)